freva-rest/entrypoint.sh

```diff
@@ -11,17 +11,17 @@ NC='\033[0m'
 
 display_logo() {
     echo -e "${BLUE}"
-    echo "░▒▓████████▓▒░▒▓███████▓▒░░▒▓████████▓▒░▒▓█▓▒░░▒▓█▓▒░░▒▓██████▓▒░ "
-    echo "░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░"
-    echo "░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░       ░▒▓█▓▒▒▓█▓▒░░▒▓█▓▒░░▒▓█▓▒░"
-    echo "░▒▓██████▓▒░ ░▒▓███████▓▒░░▒▓██████▓▒░  ░▒▓█▓▒▒▓█▓▒░░▒▓████████▓▒░"
-    echo "░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░        ░▒▓█▓▓█▓▒░ ░▒▓█▓▒░░▒▓█▓▒░"
-    echo "░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░        ░▒▓█▓▓█▓▒░ ░▒▓█▓▒░░▒▓█▓▒░"
-    echo "░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓████████▓▒░  ░▒▓██▓▒░  ░▒▓█▓▒░░▒▓█▓▒░"
+    echo -e "${BLUE}░▒▓████████▓▒░▒▓███████▓▒░░▒▓████████▓▒░▒▓█▓▒░░▒▓█▓▒░░▒▓██████▓▒░ ${NC}"
+    echo -e "${BLUE}░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░░▒▓█▓▒░${NC}"
+    echo -e "${BLUE}░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░       ░▒▓█▓▒▒▓█▓▒░░▒▓█▓▒░░▒▓█▓▒░${NC}"
+    echo -e "${BLUE}░▒▓██████▓▒░ ░▒▓███████▓▒░░▒▓██████▓▒░  ░▒▓█▓▒▒▓█▓▒░░▒▓████████▓▒░${NC}"
+    echo -e "${BLUE}░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░        ░▒▓█▓▓█▓▒░ ░▒▓█▓▒░░▒▓█▓▒░${NC}"
+    echo -e "${BLUE}░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓█▓▒░        ░▒▓█▓▓█▓▒░ ░▒▓█▓▒░░▒▓█▓▒░${NC}"
+    echo -e "${BLUE}░▒▓█▓▒░      ░▒▓█▓▒░░▒▓█▓▒░▒▓████████▓▒░  ░▒▓██▓▒░  ░▒▓█▓▒░░▒▓█▓▒░${NC}"
     echo -e "${NC}"
-    echo -e "${GREEN}=======================================================${NC}"
-    echo -e "${YELLOW}                Starting FREVA Services                 ${NC}"
-    echo -e "${GREEN}=======================================================${NC}"
+    echo -e "${GREEN}==================================================================${NC}"
+    echo -e "${YELLOW}                    Starting FREVA Services                      ${NC}"
+    echo -e "${GREEN}==================================================================${NC}"
```

Comment:
I like this a lot, but would it be possible to make this appear not blurry? I mean shadow or whatever what this is supposed to be in only one direction? Something like this:
```console
████████▓▒
█▓▒
█▓▒
██████▓▒
█▓▒
█▓▒
█▓▒

```